core/include/gnuradio-4.0/Transactions.hpp

```diff
@@ -77,8 +87,9 @@ class CtxSettings : public SettingsBase {
     std::set<std::string, std::less<>>                                                         _allWritableMembers{};   // all `isWritableMember` class members
     std::map<pmtv::pmt, std::set<std::string, std::less<>>, PMTCompare>                        _autoUpdateParameters{}; // for each SettingsCtx.context auto updated members are store separately
     std::set<std::string, std::less<>>                                                         _autoForwardParameters{};
-    MatchPredicate                                                                             _matchPred = nullMatchPred;
-    pmtv::pmt                                                                                  _activeCtx = "";
+    MatchPredicate                                                                             _matchPred        = nullMatchPred;
+    pmtv::pmt                                                                                  _activeCtx        = "";
+    settings::AutoUpdatePolicy                                                                 _autoUpdatePolicy = settings::AutoUpdatePolicy::AddToStored;
```

Comment:
This is not accessible via a public API. We should discuss whether this should be part of the block configuration and/or eventually dropped.